swagger-backend.yaml

```diff
@@ -0,0 +1,184 @@
+---
+swagger: '2.0'
+info:
+  description: Flotta backend API for remote data storage
+  title: Flotta backend API
+  version: 1.0.0
+schemes:
+  - https
+consumes:
+  - application/json
+produces:
+  - application/json
+basePath: /api/flotta-backend/v1
+definitions:
+  error:
+    type: object
+    required:
+      - message
+    properties:
+      message:
+        type: string
+  device-configuration-response:
+    type: object
+    properties:
+      message:
+        type: string
```

Comment:
pls add description for the properties.
it will be easier to understand what each `*-response` returns

Reply:
Understood

Reply:
Done.

Reply:
Thank you. this makes the API very clear :+1: